docs/introduction.md

```diff
@@ -138,6 +140,43 @@ ncn-m001# sat bash
 (CONTAINER_ID) sat-container# sat status
 ```
 
+## SAT in CSM
+
+In CSM 1.3 and newer, the `sat` command is automatically available on all the Kubernetes NCNs, though it is still
+possible to install SAT as an independent product stream. Installing the SAT product stream results in additional
+supporting components being installed.
+
+- Installing the SAT product stream creates an entry for SAT in the `cray-product-catalog` Kubernetes ConfigMap.
+  Without installing the SAT product stream, there will be no entry for this version of SAT in the output of
+  `sat showrev`.
+
+- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
+  provides uninstall and activate functionality when used with the `prodmgr` command. It is still possible to uninstall
+  previous versions of the SAT product stream.
+
+- The `sat-config-management` git repository in Gitea (VCS) and thus the SAT layer of NCN CFS configuration is
+  only available with the full SAT product stream. See [SAT CFS Configuration](#sat-cfs-configuration-details) below.
+
+- The `docs-sat` RPM package is only available with the full SAT product stream.
+
+### SAT CFS Configuration Details
+
+When the SAT product stream has not been installed, there will be no configuration content for SAT in VCS. Therefore,
```

Comment:
I have a few suggestions to help the flow of the rest of this information:
```suggestion
If the SAT product stream is not installed, there will be no configuration content for SAT in VCS. Therefore,
```

Reply:
Addressed in 45fae7c23fd959bdca55482f63824e0422d6509c